src/procedures/proc_list_indexes.c

```diff
@@ -56,6 +58,12 @@ static void _process_yield
 			continue;
 		}
 
+		if(strcasecmp("options", yield[i]) == 0) {
```

Comment:
I think 'options' is too generic of aname, as we're dealing with vector index options, I think the name should reflect it, consider renaming to 'HNSW_config'.

Reply:
this is for any options if we will introduce more options for the other index type you suggest to add more variables to the result? for example range_options and fulltext_options?

Reply:
So this is general options for all index configuration ?